src/common-rest-api/src/main/java/org/niis/xroad/restapi/auth/securityconfigurer/StaticAssetsWebSecurityConfig.java

```diff
@@ -50,6 +50,14 @@ protected void configure(HttpSecurity http) throws Exception {
                         "/js/**",
                         "/fonts/**")
                 .and()
+            .headers()
+                    .contentSecurityPolicy(
```

Comment:
nitpick: indentation seems off, compared to other blocks

Reply:
Another take on the indentation: abf32ba